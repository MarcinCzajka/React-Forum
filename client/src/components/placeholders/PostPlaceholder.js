import React from 'react';
import { Placeholder } from "semantic-ui-react";

class PostPlaceholder extends React.Component {

    render() {
        return (
            <Placeholder>
                <Placeholder.Header image>
                <Placeholder.Line />
                </Placeholder.Header>
                <Placeholder.Paragraph>
                <Placeholder.Line />
                <Placeholder.Line />
                </Placeholder.Paragraph>
            </Placeholder>
        )
    }

}

export default PostPlaceholder