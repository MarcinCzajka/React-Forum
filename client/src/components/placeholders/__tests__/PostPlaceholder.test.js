import React from 'react';
import { create } from "react-test-renderer";
import PostPlaceholder from '../PostPlaceholder';

describe('Captcha component', () => {
    test('match snapshot', () => {
        const component = create(<PostPlaceholder />);
        expect(component).toMatchSnapshot();
    });

});