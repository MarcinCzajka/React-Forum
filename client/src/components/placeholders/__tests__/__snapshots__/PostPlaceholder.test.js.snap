// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captcha component match snapshot 1`] = `
<div
  className="ui placeholder"
>
  <div
    className="image header"
  >
    <div
      className="line"
    />
  </div>
  <div
    className="paragraph"
  >
    <div
      className="line"
    />
    <div
      className="line"
    />
  </div>
</div>
`;
